cli/cli.ts

```diff
@@ -4104,7 +4106,8 @@ function buildCoreAsync(buildOpts: BuildCoreOptions): Promise<pxtc.CompileResult
                 }
             });
 
-            reportDiagnostics(res.diagnostics);
+            if (!config.partial)
```

Comment:
so no problems with this

Reply:
though, it would be probably good to report errors just before exit(1) regardless of partial

Reply:
displaying error on exit now